core/src/builder/stand_alone.rs

```diff
@@ -207,6 +207,10 @@ where
 		self.imports.clone()
 	}
 
+	pub fn commit_storage_changes(&mut self, changes: StorageChanges<B::State, Block>, at: BlockId<Block>) -> Result<(), ()> {
+		self.builder.commit_storage_changes(changes, at)
+	}
+
```

Comment:
I would actually not expose this one here and only expose the api from the core builder. To complex at the moment ^^

Reply:
Yup, I also thought the same, just wanted to play around with this stuff.